### ISSUE TITLE
[Feature Request]: KB Shortcut for navigating left menu

### ISSUE THREAD

**Issue by lewgrant** [FreeTubeApp/FreeTube#5690]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

There is no quick/easy way to use the left navigation bar with the keyboard. you can tab to them but only after tabbing through the entirety of the current screen.

### Proposed Solution

KB Shortcut. Something like Ctrl + Shift followed by a number corresponding to the nav item. ie 1 for Subscription, 2 for channels, 3 for Trending etc.

Anything would be appreciated

### Alternatives Considered

Anything else that can be done in a single KB action - like modifiers + S for subscription, C for channels etc

### Issue Labels

new keyboard shortcut

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of #2691
